ra-tls/Makefile

```diff
@@ -64,14 +60,21 @@ $(LIBDIR)/libwolfssl.a: $(LIBDIR) wolfssl
 	cd wolfssl && $(MAKE) install
 
 wolfssl: WOLFSSL_CONFIGURE_FLAGS := --prefix=$(shell readlink -f $(PREFIX)) --enable-writedup --enable-static --enable-keygen --enable-certgen --enable-certext --with-pic --disable-examples --disable-crypttests --enable-aesni --enable-tlsv10 $(WOLFSSL_CONFFLAGS)
+
 wolfssl:
-	[ ! -d wolfssl ] && git clone -b v4.6.0-stable https://github.com/wolfSSL/wolfssl && \
-	  cd wolfssl && git apply ../patch/wolfssl.patch && ./autogen.sh && \
-	  CFLAGS="$(CFLAGS)" ./configure $(WOLFSSL_CONFIGURE_FLAGS) || true
+	if [ -d "./wolfssl" ]; then \
+		cd wolfssl && ./autogen.sh && CFLAGS="$(CFLAGS)" ./configure $(WOLFSSL_CONFIGURE_FLAGS) || true; \
+	fi
```

Comment:
Not sure what this is doing.

Reply:
wolfssl must be configured before compiling. Because compile once, the configuration will be wrote into options.h, compiling again, the options will be unchanged. In other words, I compiled with RATLS_ECDSA, the Macro will be wrote into the .h file, even I compile epid, the RATLS_ECDSA is still there. So I have to configure wolfssl again against the error.

Reply:
Plz add your above comments to Makefile.

Reply:
ok, thanks

Reply:
Placing the comments in the shell text area is not working. Plz move them ahead of `wolfssl:` line. In addition, it is required to add a space between '#' and comment text.